src/main/java/edu/ucsb/courses/documents/Section.java

```diff
@@ -0,0 +1,392 @@
+        return this;
+    }
+
+    public Section departmentApprovalRequired(boolean departmentApprovalRequired) {
+        this.departmentApprovalRequired = departmentApprovalRequired;
+        return this;
+    }
+
+    public Section instructorApprovalRequired(boolean instructorApprovalRequired) {
+        this.instructorApprovalRequired = instructorApprovalRequired;
+        return this;
+    }
+
+    public Section restrictionLevel(String restrictionLevel) {
+        this.restrictionLevel = restrictionLevel;
+        return this;
+    }
+
+    public Section restrictionMajor(String restrictionMajor) {
+        this.restrictionMajor = restrictionMajor;
+        return this;
+    }
+
+    public Section restrictionMajorPass(String restrictionMajorPass) {
+        this.restrictionMajorPass = restrictionMajorPass;
+        return this;
+    }
+
+    public Section restrictionMinor(String restrictionMinor) {
+        this.restrictionMinor = restrictionMinor;
+        return this;
+    }
+
+    public Section restrictionMinorPass(String restrictionMinorPass) {
+        this.restrictionMinorPass = restrictionMinorPass;
+        return this;
+    }
+
+    public Section concurrentCourses(List<String> concurrentCourses) {
+        this.concurrentCourses = concurrentCourses;
+        return this;
+    }
+
+    public Section timeLocations(List<TimeLocation> timeLocations) {
+        this.timeLocations = timeLocations;
+        return this;
+    }
+
+    public Section instructors(List<Instructor> instructors) {
+        this.instructors = instructors;
+        return this;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == this)
+            return true;
+        if (!(o instanceof Section)) {
+            return false;
+        }
+        Section section = (Section) o;
+        return Objects.equals(enrollCode, section.enrollCode) && Objects.equals(section, section.section) && Objects.equals(session, section.session) && Objects.equals(classClosed, section.classClosed) && Objects.equals(courseCancelled, section.courseCancelled) && Objects.equals(gradingOptionCode, section.gradingOptionCode) && Objects.equals(enrolledTotal, section.enrolledTotal) && Objects.equals(maxEnroll, section.maxEnroll) && Objects.equals(secondaryStatus, section.secondaryStatus) && departmentApprovalRequired == section.departmentApprovalRequired && instructorApprovalRequired == section.instructorApprovalRequired && Objects.equals(restrictionLevel, section.restrictionLevel) && Objects.equals(restrictionMajor, section.restrictionMajor) && Objects.equals(restrictionMajorPass, section.restrictionMajorPass) && Objects.equals(restrictionMinor, section.restrictionMinor) && Objects.equals(restrictionMinorPass, section.restrictionMinorPass) && Objects.equals(concurrentCourses, section.concurrentCourses) && Objects.equals(timeLocations, section.timeLocations) && Objects.equals(instructors, section.instructors);
```

Comment:
We should ideally use `EqualsBuilder` to reduce the number of tests we need to do.

Reply:
Yes... I'm moving to that one repo at a time.